holidays/countries/denmark.py

```diff
@@ -39,7 +39,9 @@ def _populate(self, year):
         self[easter_date + td(days=-2)] = self.tr("Langfredag")
         self[easter_date] = self.tr("Påskedag")
         self[easter_date + td(days=+1)] = self.tr("Anden påskedag")
-        self[easter_date + td(days=+26)] = self.tr("Store bededag")
+        # See https://www.ft.dk/samling/20222/lovforslag/l13/index.htm
+        if year < 2024:
```

Comment:
```suggestion
        if year <= 2023:
```
Just for code style consistency.